jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -115,11 +120,18 @@ public JvmServiceImpl(final JvmPersistenceService jvmPersistenceService,
         this.binaryDistributionService = binaryDistributionService;
         this.binaryDistributionLockManager = binaryDistributionLockManager;
         this.historyFacadeService = historyFacadeService;
+        this.webServerPersistenceService = webServerPersistenceService;
         this.fileUtility = fileUtility;
     }
 
 
     protected Jvm createJvm(final CreateJvmRequest aCreateJvmRequest) {

```

Comment:
I can't help notice that we have a createJvm method in this service. The purpose of the JvmStateService is to provide state related functionality. Can you ask @akinny why it's here ? I would prefer it to be in JvmService.